### ISSUE TITLE
How can I create alarm dialog with no cancel button?

### ISSUE THREAD

**Issue by lexer** [visionmedia/uikit#41]
Should I make my own implementation like Confirmation dialog, or it is possible to reuse confirmation dialog for this purpose?

**Comment by tj** [comment]
hmmm yeah maybe, or use the notifications maybe?

**Comment by lexer** [comment]
In our app we need to show modal window that will block user until he read the message and press "Ok"

**Comment by lexer** [comment]
Another example. User should be blocked until he will receive response via push notification. If user don't want to wait he ca press single button "Cancel" and resume his work.

**Comment by lexer** [comment]
May be it would be better to modify confirmation to show ok and cancel only if they are declared. This is how android Alerts are designed for example.

new ConfirmationDialog().ok("done") - this is alarm dialog with single ok button
new ConfirmationDialog().cancel("reset") - this is alarm dialog with single cancel button.

However in this design it should be named smth like "AlarmDialog" that is more generic name.

Have you also thought about separate callbacks? .ok("done", function(){ });

**Comment by lexer** [comment]
I mean "AlertDialog" not alarm. We will try to make a pull request today.

**Comment by tj** [comment]
yeah we could have an Alert and `ui.alert(msg)` with .modal() etc default. modals in generate are kinda bad UX but I can see in some cases if you have a critical issue that needs attention

**Comment by lexer** [comment]
yep sure. modal should be optional.

**Comment by lexer** [comment]
We have added pull request https://github.com/visionmedia/uikit/pull/42
